app/controllers/articles_controller.rb

```diff
@@ -20,49 +18,36 @@ def new
 
   # GET /articles/1/edit
   def edit
+    authorize @article
   end
 
   # POST /articles
-  # POST /articles.json
   def create
     @article = Article.new(article_params)
 
-    respond_to do |format|
-      if @article.save
-        format.html {
-          redirect_to @article, notice: 'Article was successfully created.'
-          current_user.articles << @article
-        }
-        format.json { render :show, status: :created, location: @article }
-      else
-        format.html { render :new }
-        format.json { render json: @article.errors, status: :unprocessable_entity }
-      end
+    authorize @article
+    if @article.save
+      redirect_to @article, notice: 'Article was successfully created.'
+      current_user.articles << @article
+    else
+      render :new
     end
   end
 
   # PATCH/PUT /articles/1
-  # PATCH/PUT /articles/1.json
   def update
-    respond_to do |format|
-      if @article.update(article_params)
-        format.html { redirect_to @article, notice: 'Article was successfully updated.' }
-        format.json { render :show, status: :ok, location: @article }
-      else
-        format.html { render :edit }
-        format.json { render json: @article.errors, status: :unprocessable_entity }
-      end
+    authorize @article
+    if @article.update(article_params)
+      redirect_to @article, notice: 'Article was successfully updated.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.